src/fake_latch_plugin.cpp

```diff
@@ -0,0 +1,113 @@
+#include <nav2_gazebo/fake_latch_plugin.hpp>
+#include <ignition/gazebo/Model.hh>
+#include <ignition/plugin/Register.hh>
+#include <ignition/gazebo/components.hh>
+#include <ignition/gazebo/components/DetachableJoint.hh>
+
+namespace nav2_gazebo
+{
+
+    FakeLatchPlugin::FakeLatchPlugin()
+    : latched_{false}
+    , unlatch_requested_{false}
+    , robot_name_{""}
+    , cart_name_{""}
+    , robot_{ignition::gazebo::kNullEntity}
+    , cart_{ignition::gazebo::kNullEntity}
+    , latch_joint_{ignition::gazebo::kNullEntity}
+    {
+        ignwarn << "Initializing FakeLatch plugin" << std::endl;
```

Comment:
Get rid of this? Might be more annoying than helpful.

Reply:
Yeah this can be removed